pom.xml

```diff
@@ -8,18 +8,16 @@
   <packaging>pom</packaging>
   <version>7-201708-EA-SNAPSHOT</version>
   <properties>
-    <!-- Spring Boot / Cloud / Stream deps -->
-    <spring.boot.version>2.0.0.M3</spring.boot.version>
-    <spring.cloud.version>Finchley.BUILD-SNAPSHOT</spring.cloud.version>
-    <spring.cloud.stream.version>Elmhurst.BUILD-SNAPSHOT</spring.cloud.stream.version>
-    <!-- using SNAPSHOT version to benefit from https://github.com/spring-projects/spring-hateoas/issues/583 -->
-    <spring.hateoas.version>0.24.0.BUILD-SNAPSHOT</spring.hateoas.version>
-
     <!--configuration properties-->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <enforcer.failOnDuplicatedClasses>true</enforcer.failOnDuplicatedClasses>
     <jdk.version>1.8</jdk.version>
+    <docker.keycloak.port>8180</docker.keycloak.port>
```

Comment:
I thought that all services would use the same configuration. This should be the case of at least timeouts. I'd like to avoid declaring them for every service application.